pommerman/forward_model.py

```diff
@@ -499,14 +499,30 @@ def make_bomb_maps(position):
             ''' Makes an array of an agents bombs and the bombs attributes '''
             blast_strengths = np.zeros((board_size, board_size))
             life = np.zeros((board_size, board_size))
+            moving_direction = np.zeros((board_size, board_size))
 
             for bomb in bombs:
                 x, y = bomb.position
                 if not is_partially_observable \
                    or in_view_range(position, x, y):
                     blast_strengths[(x, y)] = bomb.blast_strength
                     life[(x, y)] = bomb.life
-            return blast_strengths, life
+                    if bomb.moving_direction is not None:
+                        moving_direction[(x, y)] = bomb.moving_direction.value
+                    else:
```

Comment:
nit: not needed.

Reply:
It currently is because the moving direction is initialized to None by default. I'll change that to 0 and remove the if else case.

```python
class Bomb(object):
    """Container for the Bomb object."""

    def __init__(self,
                 bomber,
                 position,
                 life,
                 blast_strength,
                 moving_direction=None):
        self.bomber = bomber
        self.position = position
        self.life = life
        self.blast_strength = blast_strength
        self.moving_direction = moving_direction
```

Reply:
sorry! i mean just the `else` clause because you initialized it above to 0.